3p/3p.js

```diff
@@ -249,16 +249,16 @@ export function validateData(data, mandatoryFields, opt_optionalFields) {
  * @param {!Array<string>} alternativeFields
  */
 function validateExactlyOne(data, alternativeFields) {
-  let countFileds = 0;
+  let countFields = 0;
 
   for (let i = 0; i < alternativeFields.length; i++) {
```

Comment:
Can make this a one liner:
user().assert(alternativeFields.filter(field => data[field]).length === 1, ...);

Reply:
On it. This is actually not my code though, I just stumbled upon this file randomly and my eyes fell on the typo :smile: